pkg/generate.go

```diff
@@ -370,7 +377,7 @@ func generateRoute(options gitopsv1alpha1.GeneratorOptions) *routev1.Route {
 			APIVersion: "route.openshift.io/v1",
 		},
 		ObjectMeta: v1.ObjectMeta{
-			Name:      options.Name,
+			Name:      routeName,
```

Comment:
Is this the only place that needs to be changed?  See also line ~390 or ~393

Also do we need to consider the other resources, eg the names of the other deployment, deployment patch and service resources?

Reply:
No, this is specific to the route's metadata.name, as the generated host name is derived from it. The other resources are fine with the default name.